app/validators/initiative_has_manager_validator.rb

```diff
@@ -0,0 +1,7 @@
+class InitiativeHasManagerValidator < ActiveModel::Validator
```

Comment:
I know Rails supports this BUT the intention behind it is to do custom validations that can be reused in multiple models. This is not the case for this validation as it's intrinsically linked to Initiatives model and it validates a a relationship that no other entity will have

Reply:
Super fair, I'll move it to a private method